src/pages/index.tsx

```diff
@@ -1,14 +1,256 @@
+    },
+    {
+      path: 'http://www.esd.whs.mil/DD/index.html',
+      heading: 'DoD Directives and Issuances',
+      text: 'Find all the published DoD directives, forms and issuances.',
+    },
+    {
+      path: 'https://dots.dodiis.mil/',
+      heading: 'DoDIIS One-way Transfer Service (DOTS)',
+      text: 'Send files from a lower classification network to a higher classification network.',
+    },
+    {
+      path: 'https://www.intelink.gov',
+      heading: 'Intelink.gov',
+      text: 'Intelligence and data: access all the resources of the intelligence community.',
+    },
+    {
+      path: 'https://www.milsuite.mil',
+      heading: 'milSuite',
+      text: 'Collaborate and create with the milSuite tools.',
+    },
+  ]
+
+  const physicalAssessment: AnnouncementCardProps = {
+    heading: 'Physical fitness assessments will resume July 1st 2021',
+    tag: 'news',
+    bgColor: 'gradient--orange bg-accent-warm-dark',
+    cols: true,
+    path: 'https://www.spaceforce.mil/News/Article/2525699/pt-test-pushed-to-july-updates-to-scoring-physical-components-ahead/',
+  }
+
+  const videoCSO: AnnouncementCardProps = {
+    heading: "Video from the Chief of Space Operations's latest townhall",
+    tag: 'news',
+    bgColor: 'gradient--orange bg-accent-warm-dark',
+    cols: true,
+    path: 'https://www.milsuite.mil/video/watch/video/40363',
+  }
+
   return (
     <>
-      <Logo />
-      USSF Portal
-      <LinkTo href="/test-page">A test page</LinkTo>
+      <CovidSiteAlert />
+      <section
+        className={`usa-section padding-top-4 padding-bottom-0 ${styles.home}`}>
+        <div className="grid-container">
+          <div className="grid-row grid-gap">
+            <div className="tablet:grid-col-8 usa-prose">
+              <CardGroup>
+                <AnnouncementCard {...physicalAssessment} />
+                <AnnouncementCard {...videoCSO} />
+              </CardGroup>
+            </div>
+            <div className="tablet:grid-col-4">
+              <div className="quick-links usa-prose border-top padding-top-2">
+                <h2 className="display-inline-block text-normal font-heading-md">
+                  Quick Links
+                </h2>
+                <ul className="usa-list usa-list--unstyled margin-top-2 font-body-xs">
+                  <li>
+                    <a href="https://mypay.dfas.mil/">myPay</a>
+                  </li>
+                  <li className="margin-top-05">
+                    <a href="https://dtsproweb.defensetravel.osd.mil/dts-app/pubsite/all/view/">
+                      DTS
+                    </a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section
+        className={`usa-section padding-top-6 padding-bottom-0 bg-base-lightestx ${styles.home}`}>
+        <div className="grid-container usa-prose">
```

Comment:
@suzubara I noticed the Grid components in your About page PR and probably should've used it in mine... 🙈 Still getting used to the React USWDS components that are available.

Reply:
no problem, I like to use them where I can but they also don't fit every scenario so I wouldn't worry too much about it!